client/src/client.js

```diff
@@ -86,7 +86,28 @@ function line_operations(a_line) {
  ***********************************************************/
 async function two_lines() {
   // calling our lambda function
-  var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
+  //var cr_call;
+  try {
+    var cr_call = await axios.get('https://91duv1eln6.execute-api.us-east-1.amazonaws.com/dev/chartdb');
```

Comment:
Best practice is to use let (or in this case const) instead of var.

Reply:
I'm having trouble with this one.  If I change this to const or let, they both go into catch and can't identify the variable later in the function.  I tried to change the way I was doing the axios get request.  Using .then and .catch, but I ran into trouble that way too.